packages/styles/gulpfile.js

```diff
@@ -194,7 +209,8 @@ exports.default = gulp.task(
     'prebuild-env-vars',
     gulp.parallel(
       gulp.series('map-icons', 'copy', 'autoprefixer', 'transform-package-json'),
-      gulp.series('temporarily-copy-token-files', 'sass'),
+      gulp.series('temporarily-copy-token-files', 'temporarily-copy-icon-files', 'sass'),
     ),
+    'copy-icon-files-to-dist',
```

Comment:
Only try this, if you run `copy-icon-files-to-dist` as a subtask of the `copy` task (mentioned above).
It would be good to ensure, we first copy the temp files (tokens/icons) to the ./src folder and then copy things from ./src to ./dist, before we finally build our sass files.

```js
exports.default = gulp.task(
  'build',
  gulp.series(
    'prebuild-env-vars',
    gulp.parallel(
      gulp.series('map-icons', 'autoprefixer', 'transform-package-json'),
      gulp.series('temporarily-copy-token-files', 'temporarily-copy-icon-files', 'copy', 'sass'),
    ),
  ),
);
```